TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java

```diff
@@ -59,6 +59,11 @@ public void configureTeleOpBindings() {
        *    Go to start from outtaking -> down d-pad
        */
 
+
+
+//TODO to be determined: operator command override button
```

Comment:
This is very picky, but it's nice to have a colon after the TODO. 
```suggestion
//TODO: to be determined: operator command override button
```